### DISCUSSION TITLE
Open to Suggestion?

### DISCUSSION THREAD

**Discussion by eshoberg** [leoafarias/sidekick#253] [General]
Team up with **@syncfusion / flutter-widgets UI** help and the two of you build a more robust set of tools. 

Honestly I was kind of excited about Flutter until I went through the install (from sources) and using your app as well and figured out that no-one wrote a built in GUI for the awesome tools they built. 

[Seems to be some want for something.](https://github.com/flutter/flutter/issues/17548)

**Comment by eshoberg** [comment]
> Team up with **@syncfusion / flutter-widgets UI** help and the two of you build a more robust set of tools.
> 
> Honestly I was kind of excited about Flutter until I went through the install (from sources) and using your app as well and figured out that no-one wrote a built in GUI for the awesome tools they built.
> 
> [Seems to be some want for something.](https://github.com/flutter/flutter/issues/17548)

Well, I'd only known about SyncFusions Windows App that has demos of a bunch of Flutter UI components. I guess SyncFusion has a whole plethora of graphical and non-grafical IDE UI tools for all kinds of platforms, including Flutter. (I actually should have known this, I believe I was going to look in to them before but was never able to make a full follow up. - We're not all millionaires, and this IDE||UI software is expensive. But perhaps that's why people should take another look. 

I'd have to say though, Sidekick peeps, there is likely a market out there for a UI||IDE that updates the SDK versions from underneath. (All you got to do is just make sure you are very specific about how that trick is performed. (I.E. With environment variables.)) 

||- I did/do enjoy the usefulness of your app. Thanks for creating it.

~ V

**Reply by leoafarias** [reply]
Thanks for the suggestion. I knew a team that was working on something like that. However an IDE would be outside of the scope of Sidekick.
